buildenv/jenkins/variables/defaults.yml

```diff
@@ -194,6 +194,59 @@ ppc64le_linux_xl:
     13:
       ? special.system
 #========================================#
+# Linux PPCLE 64bits Compressed Pointers /w JITSERVER
+#========================================#
+ppc64le_linux_jit:
+  boot_jdk:
+    8: '/usr/lib/jvm/adoptojdk-java-80'
+    11: '/usr/lib/jvm/adoptojdk-java-11'
+    12: '/usr/lib/jvm/adoptojdk-java-11'
+    13: '/usr/lib/jvm/adoptojdk-java-12'
+    next: '/usr/lib/jvm/adoptojdk-java-13'
+  release:
+    8: 'linux-ppc64le-normal-server-release'
+    11: 'linux-ppc64le-normal-server-release'
+    12: 'linux-ppc64le-server-release'
+    13: 'linux-ppc64le-server-release'
+    next: 'linux-ppc64le-server-release'
+  freemarker: '/home/jenkins/freemarker.jar'
+  openjdk_reference_repo: '/home/jenkins/openjdk_cache'
+  node_labels:
+    build:
+      8: 'ci.role.build && hw.arch.ppc64le && sw.os.ubuntu'
+      11: 'ci.role.build && hw.arch.ppc64le && sw.os.ubuntu'
+      12: 'ci.role.build && hw.arch.ppc64le && sw.os.ubuntu'
+      13: 'ci.role.build && hw.arch.ppc64le && sw.os.ubuntu'
+      next: 'ci.role.build && hw.arch.ppc64le && sw.os.ubuntu'
+  extra_getsource_options:
+    8: '--openssl-version=1.1.1d'
+    11: '--openssl-version=1.1.1d'
+    12: '--openssl-version=1.1.1d'
+    13: '--openssl-version=1.1.1d'
+    next: '--openssl-version=1.1.1d'
+  extra_configure_options:
+    8: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    11: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    12: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    13: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    next: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+  build_env:
+    vars:
+      8: 'CC=gcc-7 CXX=g++-7'
+      11: 'CC=gcc-7 CXX=g++-7'
+      12: 'CC=gcc-7 CXX=g++-7'
+      13: 'CC=gcc-7 CXX=g++-7'
+      next: 'CC=gcc-7 CXX=g++-7'
+  excluded_tests:
+    13:
```

Comment:
Although this is JitServer and not Java, please excluded 8 and 11 here as well, just in case. We don't have the machines to add any more special.system testing in the nightly builds.

Reply:
Unless we add/request a jit spec in the nightly (or any build that runs special.system), we won't have to worry about this.

Reply:
Understood, it is just weird to have a copy of some excludes. Either all should be excluded, or the section removed for the JitServer specs. I expect we can fix is up at some point in relation to #7809.